astropy/io/fits/tests/test_fitsheader.py

```diff
@@ -0,0 +1,159 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import sys
+
+import numpy as np
+import re
+
+from . import FitsTestCase
+from ..hdu import PrimaryHDU
+from ..scripts import fitsheader
+from ....tests.helper import catch_warnings
+from ....tests.helper import pytest
+from ....utils.exceptions import AstropyDeprecationWarning
+from ....version import version
+ 
+# @pytest.fixture(scope=module , params=['./tests/test0.fits'])
+# def resource_testfile_setup(request):
+#     with open(request.params[0]) as test_fits:
+#         yield test_fits
+
+@pytest.fixture(scope='module')
```

Comment:
Yes. module scope fixtures distribute the return value across functions which need it, don't they? Although I know it's almost no overhead. Should I remove it and replace by functions?